src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java

```diff
@@ -36,6 +39,20 @@ public Map<String, SubmissionField> prepareSubmissionFields(Submission submissio
         results.put("applicantSSN", new SingleField("applicantSSN", SubmissionUtilities.formatSSN((String) inputData.get("encryptedSSN")), null));
         results.put("speaksEnglish", new SingleField("speaksEnglish", (String) "Yes", null));
 
+        prepareAnswersToSensitiveQuestions(inputData, results);
         return results;
     }
+
+    private static void prepareAnswersToSensitiveQuestions(Map<String, Object> inputData, Map<String, SubmissionField> results) {
+        for (String question : SetDefaultSensitiveConvictionAnswers.SENSITIVE_CONVICTION_QUESTIONS) {
```

Comment:
Pulling these from a specific Action feels weird, but maybe having as an enum feels weird as well.

Reply:
Agreed, I couldn't think of a proper place to put it.